app/web/features/communities/events/EventTimeChanger.tsx

```diff
@@ -153,16 +124,17 @@ export default function EventTimeChanger({
                 t("communities:past_time_error")
               );
             },
-          })}
-          defaultValue={eventStartTime || null}
```

Comment:
Is defaultValue being passed in the field now?

For example if you go to the `EditEvent` component (where this component is also used) and it already has a default start and end time, are you seeing them passed through when you go to edit?